### PR TITLE
Created name.txt file

### PR DESCRIPTION
**Pull request by Neaz-Mahmood**
Added name.txt file

### CODE REVIEW

File: name.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Neaz Mahmood
```
Comment (BackAged) [inline name.txt:1]: @Neaz-Mahmood Are you out of your sane mind? what's this?

**Review by BackAged — CHANGES_REQUESTED** [review]

File: name.txt (hunk 1 above)
Comment (Neaz-Mahmood) [reply name.txt:1]: What is the problem?

**Review by BackAged — APPROVED** [review]
Looks good!

**Review by AlishaZamanKhan — APPROVED** [review]
Looks Good

**Review by noyon57 — APPROVED** [review]
well done!

**Pull request merged**